testgrid/config.yaml

```diff
@@ -7548,6 +7548,18 @@ dashboards:
     test_group_name: arm64-conformance-aws-cluster
     description: Runs conformance tests by using kubetest against latest version of kubernetes on AWS"
 
+# EKS dashboard
+- name: conformance-eks
```

Comment:
Not sure I agree. Please see jobs under https://testgrid.k8s.io/conformance-all. There are two categories of them:

- "dev" (running against release branch tips)
- "release" (running against latest patch release)

These jobs clearly belong to the second category. Am I missing sth?

Reply:
release means tagged as in the latest 1.X.Y where Y changes regularly, as opposed to the latest commit on 1.X branch, it's still a CI build

Reply:
IE "release" is not `1.X.<arbitrary-patch>`, it is always `1.X.<latest>`. These are special additional signal to the preferred dev, which most of the jobs are doing.

Reply:
There are descriptions of these on each and every job entry. We've kept these as consistent as possible throughout the dashboard.

Reply:
should add: I actually need to remove some tabs from this dashboard for falling to properly post results including:
- alibaba
- baidu
- DO

Reply:
> release means tagged as in the latest 1.X.Y where Y changes regularly, as opposed to the latest commit on 1.X branch, it's still a CI build. IE "release" is not 1.X.<arbitrary-patch>, it is always 1.X.<latest>.

I meant just the same in my above comment :)

> These are special additional signal to the preferred dev, which most of the jobs are doing.

It doesn't seem like a compulsory requirement to have dev. For e.g gardener jobs don't seem to have - https://testgrid.k8s.io/conformance-gardener

In general, I intended to add these jobs because the conformance dashboard [readme](https://github.com/kubernetes/test-infra/blob/master/testgrid/conformance/README.md) clearly says `All Kubernetes cluster providers are invited to post results from their conformance test jobs`. Regarding CI signal, it only says `results from reliable continuous integration against the release branches may even be used as a signal by the Kubernetes release team in the release-blocking dashboards`. In this case, there is no intention to make those jobs release-blocking. Just want to share EKS results too in the dashboard (they're already shown under sig-aws jobs) :)

Reply:
Btw - I'm open to closing this PR if you still think it's not worthwhile.

Reply:
I see. We might need to clarify that, blocking is also the incorrect term now, they are most likely going to be "informing" now that we've re-organized release jobs.

Ideally what we're looking for in https://testgrid.k8s.io/conformance-all is the ability to quickly identify the state of those branches within at least the conformance jobs against many configurations / CP. 

The eventual intent is that all stable signal in that dashboard should be at least release "informing", which requires that they actually be against the branch. So far this is just OpenStack and GCE, but only because we've been doing a bigger refactor of the release dashboards, I will be following up on vSphere and kind at least.

A second dashboard devoted to just "how are the tests holding up against 1.X.something" might be useful, but I think it's significantly less useful, because we can also see that from the first dashboard.

Reply:
I will file another PR to clarify some of the language in the readme particularly around blocking -> informing after following up again on this with SIG-release.

In the meantime I'd appreciate help brain storming names etc. for a second dashboard devoted to "upstream client / test builds, stable cloud provider distro clusters".

We have at least one other job doing this that is only in a sig dashboard currently, and it would be nice to collect these up into another dashboard.

Reply:
That seems reasonable but, but I'd prefer to get a second name sorted so that we can distinguish, in order to keep lobbying to SIG-release straight forward. We've had some light pushback in the past on blocking on signal that does not come from the project's own CI for exmaple, but they have caught regressions and getting them into the release signal is very valuable to avoid having a mono-culture in the signal.

Reply:
Yeah, that seems fair to have another dashboard and distinguish both types of jobs. As I said offline:

- I'm not sure if we should be needing to rely on conformance jobs that are not dev (for blocking releases)
- while that other dashboard won't be useful from sig-release perspective it can still stay as a hallmark for kubernetes' *runs-everywhere* nature

That said, I'd prefer not blocking this change on that dashboard refactoring. Wdyt?

Reply:
fair enough. when we refactor we should remove the GCE release tagged (not latest commit / dev) jobs as well as discussed